docs/source/membership/membership.md

```diff
@@ -1,341 +1,194 @@
+
+Finally, consider if you want to join an _existing_ network, you need a way to turn your identity into something that is recognized by the network. The MSP is the mechanism that enables you to participate on a permissioned blockchain network. To transact on a Fabric network a member needs to:
+
+- Have an identity issued by a CA that is trusted by the network.
+- Become a member of an _organization_ that is recognized and approved by the network members. The MSP is how the identity is linked to the membership of an organization. Membership is achieved by adding the member's public key (also known as certificate, signing cert, or signcert) to the organization’s MSP.
+
+## What is an MSP?
+
+Despite its name, the Membership Service Provider does not actually provide anything. Rather, the implementation of the MSP requirement is a folder that contains a list of related identities and is used to store users's certificates or in the case of channel configuration, a list of organizations. Whereas Certificate Authorities generate the certificates that represent identities, the MSP contains a list of permissioned identities.
+
+The MSP identifies which Root CAs and Intermediate CAs are accepted to define the members of a trust domain by listing the identities of their members, or by identifying which CAs are authorized to issue valid identities for their members.
+
+But the power of an MSP goes beyond simply listing who is a network participant or member of a channel. It is the MSP that turns an identity into a **role** by identifying specific privileges an actor has on a node or channel. Note that when a user is registered with a Fabric CA, a role of admin, peer, client, orderer, or member must be associated with the user.
+
+In addition, an MSP can allow for the identification of a list of identities that have been revoked --- as discussed in the [Identity](../identity/identity.html) documentation --- but we will talk about how that process also extends to an MSP.
+
+## MSP domains
+
+MSPs occur in two domains in a blockchain network:
 
-* In channel configuration (**channel MSP**)
 * Locally on an actor's node (**local MSP**)
+* In channel configuration (**channel MSP**)
+
```

Comment:
I think we need to add "organization" here. While it is contained in a channel configuration, it is, both conceptually and functionally, a separate layer of administration. Organizations can change the parameters within its MSP at will, without needing signatures from anyone else. This is separate from channel MSPs which determine permissions at the channel level (like the consensus type, etc). 

I would bring some of the material from the "what role does an organization play in an MSP" here and define it as a separate domain of administration (using some of the language above, if you want). Then I would link down to the section where you're talking down to OUs. It's a bit disjointed, but I think it will make sense as in this section we're talking about domains (levels of administration) while the OUs is more technically information about implementation. 

Ideally this information would be before the channel MSP stuff.

Reply:
I actually had organization there and removed it due to a comment from another reviewer. But I do agree you have surfaced a gap and I like your idea. So to help tie all this together, under What role does an organization play in an MSP > Organizational Units (OUs) and MSPs> Node OU Roles and MSPs 
I added 
```**Note:** For Channel MSPs, just because an actor has the role of an administrator it doesn't mean that they can administer particular resources. The actual power a given identity has with respect to administering the system is determined by the _policies_ that manage system resources. For example, a channel policy might specify that `ORG1-MANUFACTURING` administrators, meaning identities with a role of `admin` and a Node OU of  `ORG1-MANUFACTURING`, have the rights to add new organizations to the channel, whereas the `ORG1-DISTRIBUTION` administrators have no such rights.```

Similar text is still repeated in the /admincerts folder discussion under the guise "As a reminder,.." since we can't assume people read the earlier text.